molecule/config.py

```diff
@@ -21,9 +21,8 @@
 import os
 import os.path
 
-import anyconfig
-
 from molecule import utilities
+import anyconfig
```

Comment:
Not sure why this moved? I think our standard order is:

1. Core modules
2. 3rd party modules
3. Our modules